endpoints/api/__init__.py

```diff
@@ -490,7 +490,7 @@ def wrapper(*args, **kwargs):
 
 
 def allow_if_superuser():
-    return features.SUPERUSERS_FULL_ACCESS and SuperUserPermission().can()
+    return bool(features.SUPERUSERS_FULL_ACCESS and SuperUserPermission().can())
```

Comment:
You don't really need this do you?

Reply:
the type returned ends up being `FeatureNameValue` because of the way the conditional runs. I'm setting it so the linter doesn't complain and also to be explicit